addons/mass_mailing/views/mailing_list_views.xml

```diff
@@ -216,7 +216,8 @@
                             <hr class="my-1"/>
                             <a role="menuitem" class="dropdown-item oe_kanban_action" name="action_send_mailing" type="object">New Mailing</a>
                             <hr class="my-1"/>
-                            <a role="menuitem" class="dropdown-item oe_kanban_action" name="action_archive" href="#" type="object">Archive</a>
+                            <a role="menuitem" class="dropdown-item oe_kanban_action" name="action_archive" href="#" type="object" t-if="record.active.raw_value">Archive</a>
```

Comment:
it seems there is an issue, the `t-name="menu"` element is duplicated, (there is another one a little bit above). I see that these 2 menus come from your and @creaturemusings PRs. We should only keep one, please aggregate both changes to keep what's useful between the 2 of them.